core/src/main/java/org/mskcc/cbio/oncokb/util/CacheUtils.java

```diff
@@ -541,6 +541,10 @@ public static void updateGene(Integer entrezGeneId) {
         notifyOtherServices("update", entrezGeneId);
     }
 
+    public static void forceUpdateGeneAlterations(Integer entrezGeneId) {
+        alterations.remove(entrezGeneId);
```

Comment:
If I understand correctly, this removes the old alterations record for a specific gene. But where do the new alterations got set?

Reply:
When the next time other places try to access this gene's alteration. If CacheUtil finds there is not any alteration available, it will try to recache it from database

Reply:
oh I see. Got you